server/chat-commands/info.ts

```diff
@@ -2530,6 +2530,63 @@ export const commands: ChatCommands = {
 	},
 	denyshowhelp: [`/denyshow [user] - Denies the media display request of [user]. Requires: % @ # &`],
 
+	randquote(target, room, user) {
+		room = this.requireRoom();
+		if (!room.settings.quotes?.length) return this.errorReply(`This room has no quotes.`);
+		this.runBroadcast();
+		const [quote] = Utils.shuffle(room.settings.quotes);
+		const formatted = quote.split('\n').map(item => Chat.formatText(item)).join('<br />');
+		return this.sendReplyBox(formatted);
+	},
+
+	addquote: 'quote',
+	quote(target, room, user) {
+		room = this.requireRoom();
+		target = target.trim();
+		this.checkCan('mute', null, room);
+		if (!room.settings.quotes) room.settings.quotes = [];
+		if (this.filter(target) !== target) {
+			return this.errorReply(`Invalid quote.`);
+		}
+		if (room.settings.quotes.includes(target)) {
+			return this.errorReply(`"${target}" is already quoted in this room.`);
+		}
+		if (room.settings.quotes.length >= 50) {
+			return this.errorReply(`This room already has 50 quotes, which is the maximum.`);
+		}
+		room.settings.quotes.push(target);
```

Comment:
This should probably at least track the user who added it, and the date it was added.